src/extensions/default/bramble-move-file/MoveToDialog.js

```diff
@@ -114,9 +114,7 @@ define(function (require, exports, module) {
             MoveUtils.move(source, destination)
             .done(_finishMove.bind(null, source, destination))
             .fail(_failMove.bind(null, source, destination))
-            .always(function() {
-                dialog.close();
-            });
+            .then(dialog.close());
```

Comment:
This isn't correct as you aren't calling the close function in a `then` callback, you are calling it once and passing the result (which happens to be `undefined`) into the `then` promise chain.

If this code is working the way you want it to work, then really, the fix is actually to just remove the `always` callback and not add anything else (it will have the same effect).